pkg/providers/tinkerbell/config/template-cp.yaml

```diff
@@ -296,7 +296,11 @@ spec:
             hostNetwork: true
             volumes:
             - hostPath:
+{{- if (eq .format "bottlerocket") }}
```

Comment:
I thought we weren't doing br in baremetal anymore?

Reply:
This is just for the existing Kubernetes versions. I am going to remove [this hardcoded value](https://github.com/aws/eks-anywhere-build-tooling/blob/07f3f36b77ca72faddf07b86d1fc0a58d7b93406/projects/kubernetes-sigs/cluster-api/patches/0001-Adding-capi-support-for-Bottlerocket.patch#L890) so that we can control the path from EKS-A template. So we need this change across all providers to maintain the current behavior.